### PR TITLE
Add webring links to Gabe and Adarsh

### PR DESCRIPTION
**Pull request by gabebw**
At last, the ring will be complete.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,11 @@
 <body>
 
   <div class="intro">
-    <h1>alex fox</h1>
+    <h1>
+      <a href="http://gabebw.com/" class="link">&laquo;</a>
```
Comment (gabebw) [inline index.html]: localhost 4000 2015-11-04 20-51-10

**Comment by adarsh** [comment]
ring

**Comment by foxed** [comment]
omg so beautiful. now i need to update my page!!!!!!

**Pull request merged**